Pokedex_ios/View/Controller/PokemonList/PokemonListViewController.swift

```diff
@@ -23,11 +23,21 @@ class PokemonListViewController: UIViewController {
         pokemonsTableView.dataSource = self
         setBackground()
         initViewModel()
-
+        // Setup for the advanced filter image and button
+        let advancedFilterImage = UIImage(named: "AdvancedFilter")?.withRenderingMode(.alwaysOriginal)
```

Comment:
Is better if you separate this logic on a method for better legibility. Just for thinking in the future as it will have 3 popups here